src/node_sqlite.cc

```diff
@@ -1957,6 +1957,7 @@ void StatementSync::All(const FunctionCallbackInfo<Value>& args) {
         row_values.emplace_back(val);
       }
 
+      DCHECK_EQ(row_keys.size(), row_values.size())
```

Comment:
```suggestion
      DCHECK_EQ(row_keys.size(), row_values.size());
```

Reply:
Tests are failing because of this:

```
11:47:46 ../src/node_sqlite.cc:1961:7: error: expected ';' before 'Local'
11:47:46  1961 |       Local<Object> row_obj = Object::New(
11:47:46       |       ^~~~~
11:47:46 make[2]: *** [libnode.target.mk:
```